microverse_collins_yemi-main/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="main">
+    <div>
```

Comment:
Please consider using a header tag in replace of the div tag to make it semantically correct since that section is the hearer of your page.
```suggestion
    <header>
```